structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/converter/StructuredEventToClusterShapeConverter.java

```diff
@@ -80,4 +90,28 @@
 
         return cdpClusterShape;
     }
+
+    private String getClusterTemplateDetails(String blueprintJson) {
+        JSONObject jsonObject = new JSONObject();
```

Comment:
Right now the returned object is { service : counter } map. It is not so great for extendability if in the future someone want to add other info from the template besides the services.
I'd modify this function to return a json that looks like this: { "services" : { "LIVY" : 1, "HDFS" : 1, ...} }. So basically move the current created object into the "services" field of the json. In case of an error it can be simply { "services" : {} } or { "services" : null }